iothub/service/src/FeedbackRecord.cs

```diff
@@ -7,25 +7,51 @@ namespace Microsoft.Azure.Devices
     using Newtonsoft.Json;
 
     /// <summary>
-    /// Contains properties of an individual feedback record. 
+    /// Contains properties of an individual feedback record.
     /// </summary>
     public class FeedbackRecord
     {
+        /// <summary>
+        /// The MessageId of the cloud-to-device message to which this feedback information relates.
+        /// </summary>
         [JsonProperty(PropertyName = "originalMessageId", DefaultValueHandling = DefaultValueHandling.Ignore)]
         public string OriginalMessageId { get; set; }
 
+        /// <summary>
+        /// String values for StatusCode.
+        /// </summary>
         [JsonProperty(PropertyName = "description", DefaultValueHandling = DefaultValueHandling.Ignore)]
         public string Description { get; set; }
 
+        /// <summary>
+        /// The DeviceGenerationId of the target device of the cloud-to-device message to which this piece of feedback relates.
```

Comment:
I actually know what generation Ids are.

Let's say you create a device called "foo". Later you delete that device, and then add another device named "foo".

Are they the same device? They shared the same device Id, but they may not be the same actual device.

You can distinguish them by the generation Id.

`The device generation Id distinguishes device registry instance lifetimes.` That is still hard to swallow, so I'd put in remarks based on my example above.

Reply:
Ok will do. I did see that explanation on msdn docs. Will add it to help clarify.

Reply:
I did not know about this generation Id, that'd definitely be helpful to add.